fvm/env.go

```diff
@@ -86,6 +86,8 @@ type commonEnv struct {
 	logs          []string
 	rng           *rand.Rand
 	traceSpan     opentracing.Span
+
+	frozenAccounts []common.Address
```

Comment:
Thinking about this a bit more, `frozenAccounts []common.Address` should be part of `accounts`, not part of the env.

Reply:
it's part of env right now cuz SetAccountFrozen is part of env.  I completely agree that these should be refactored out of env, and intend to do so in future PRs.